build/scripts/build_helper.spgw

```diff
@@ -215,89 +179,89 @@ check_install_spgwu_deps() {
   $SUDO $INSTALLER install $OPTION $PACKAGE_LIST
   ret=$?;[[ $ret -ne 0 ]] && return $ret
 
-
   # Libraries
   if [[ $OS_DISTRO == "ubuntu" ]]; then
     case "$(get_distribution_release)" in
       "ubuntu18.04")
         # Currently Ubuntu18.04 has boost 1.65 as the latest
         # We need at least 1.66
         # PPA has 1.67
+        $SUDO $INSTALLER install $OPTION software-properties-common
         $SUDO add-apt-repository ppa:mhier/libboost-latest --yes
         $SUDO $INSTALLER update
-        specific_packages="libconfig++-dev libboost1.67-dev"
+        specific_packages="libconfig++-dev libasio-dev libboost-all-dev libasan4"
+        ;;
+      "ubuntu20.04")
+        specific_packages="libconfig++-dev libasio-dev libboost-all-dev libasan5"
         ;;
       *)
-        specific_packages="libconfig++-dev libasio-dev libboost-all-dev"
+        specific_packages="libconfig++-dev libasio-dev libboost-all-dev libasan?"
         ;;
     esac
     # removed libspdlog-dev
     PACKAGE_LIST="\
       $specific_packages \
-      guile-2.0-dev \
       libcurl4-gnutls-dev \
```

Comment:
@rdefosse we need both libcurl4 and libcurl4-gnutls-dev ?

Reply:
@arora-sagar 

I tried to remove them and it did not compile.